src/components/treeInfo/index.tsx

```diff
@@ -71,7 +71,7 @@ const TreeInfo: React.FC<TreeProps> = ({
             // Display 'Open Planting Site' if no tree has been planted
             // Otherwise, display the tree's commonName or 'Unknown Species' if no commonName exists
             pageTitle={
-              siteData.entries[0].treePresent
+              siteData.entries[0] && siteData.entries[0].treePresent
```

Comment:
⛏️ There's an interesting JS syntax that makes this simpler, called the optional chaining operator. Could we use this here (and in `src/containers/treePage/index.tsx`)? For more info, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

```suggestion
              siteData.entries[0]?.treePresent
```